README.md

```diff
@@ -20,8 +20,14 @@ Azure Cloud Advocates at Microsoft are pleased to offer a 12-week, 24-lesson cur
 
 Get started with the following resources:
 
+ main
```

Comment:
Looks like there is some text added here and below of "main". And the links below got duplicated. I believe once those items are cleaned up we can merge this. Thanks!